src/routes/clinical-proxy.ts

```diff
@@ -102,4 +102,21 @@ router.use(
   }),
 );
 
+router.use(
+  '/program/:programId/clinical-search-results',
+  createProxyMiddleware({
+    target: CLINICAL_SERVICE_ROOT,
+    pathRewrite: (pathName: string, req: Request) => {
+      const programId = req.params.programId;
+      return urlJoin(
+        '/clinical/program/',
```

Comment:
this one here is a good pattern for using UrlJoin - separate variables from strings

Reply:
Updated